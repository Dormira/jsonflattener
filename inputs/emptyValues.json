{
	"a": 1,
	"test": "",
	"b": true,
	"c": {
		"c" : [],
		"e": {"f": {},
		      "g": {}}
	}
}
