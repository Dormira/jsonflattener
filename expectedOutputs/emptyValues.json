{
 "a": 1,
 "test": "",
 "b": true,
 "c.c": [],
 "c.e.f": {},
 "c.e.g": {}
}